### PR TITLE
Add debug for ethers lib to track Infura call 

### PR DESCRIPTION
**Pull request by trangtran-enya**

### CODE REVIEW

File: packages/batch-submitter/src/exec/run-batch-submitter.ts [hunk 1]
```diff
@@ -138,10 +138,20 @@ export const run = async () => {
     env.DEBUG_IMPERSONATE_PROPOSER_ADDRESS
   )
 
+  const setEtherDebug = (configs: { provider: StaticJsonRpcProvider }) => {
```
Comment (cby3149) [inline packages/batch-submitter/src/exec/run-batch-submitter.ts:141]: It's not the correct place. It's in `src/exec/run-batch-submitter.ts`.

File: packages/message-relayer/src/exec/withdraw.ts [hunk 2]
```diff
@@ -8,12 +8,17 @@
...
 import { predeploys, getContractInterface } from '@eth-optimism/contracts'
 import { sleep } from '@eth-optimism/core-utils'
+import { Logger, LoggerOptions } from '@eth-optimism/common-ts'
 import dotenv from 'dotenv'
 
 /* Imports: Internal */
 import { getMessagesAndProofsForL2Transaction } from '../relay-tx'
 
 dotenv.config()
+const loggerOptions: LoggerOptions = {
```
Comment (cby3149) [inline packages/message-relayer/src/exec/withdraw.ts:18]: This file isn't use in production.

**Review by boyuan-chen — CHANGES_REQUESTED** [review]

**Comment by InoMurko** [comment]
@trangtran-enya can we figure out, from the calls - who is making them?
so that we have separate dashboards for every service. this way we could figure out who's making the most requests, and than gradually figure out which rpc calls are most used by which service

File: packages/batch-submitter/src/exec/run-batch-submitter.ts (hunk 1 above)
Comment (cby3149) [reply packages/batch-submitter/src/exec/run-batch-submitter.ts:141]: My fault. You are right.

**Comment by trangtran-enya** [comment]
> The monitor also makes RPC calls to infura. The PR of adding the monitor service is on the way.

Okie, so I wait for those PRs and update afterward

**Review by boyuan-chen — APPROVED** [review]

**Comment by InoMurko** [comment]
ping @trangtran-enya @cby3149

**Comment by CAPtheorem** [comment]
stale pr - closing for now

**Pull request closed without merging**